src/openapi.yaml

```diff
@@ -6122,7 +6122,14 @@ components:
         hideNonApplicableSteps:
           type: boolean
           readOnly: true
+        cosignLoginInfo:
+          type: object
+          additionalProperties: {}
+          nullable: true
+          readOnly: true
+          description: Info about the plugin to use for co-sign authentication.
```

Comment:
the type/shape of this is not very enlightening :stuck_out_tongue: 

Probably you need to properly specify the serializers so that the API schema can be inferred correctly.